Meta/Lagom/Wasm/js_repl.cpp

```diff
@@ -119,9 +118,8 @@ static ErrorOr<bool> parse_and_run(JS::Interpreter& interpreter, StringView sour
         if (s_dump_ast)
             script_or_module->parse_node().dump(0);
 
-        if (s_run_bytecode) {
-            JS::Bytecode::Interpreter bytecode_interpreter(interpreter.realm());
-            result = bytecode_interpreter.run(*script_or_module);
+        if (auto* bytecode_interpreter = g_vm->interpreter_if_exists()) {
```

Comment:
Should this be `bytecode_interpreter_if_exists()`?

Reply:
Yes, absolutely :O